concoction-lib/src/test/java/info/mmpa/concoction/scan/model/InstructionMatchSerializationTests.java

```diff
@@ -16,7 +17,7 @@
 import static org.junit.jupiter.api.Assertions.*;
 
 /**
- * Tests for {@link InstructionMatchEntry} serialization.
+ * Tests for {@link InstructionsMatchingModel} and {@link InstructionMatchEntry} serialization.
  */
 public class InstructionMatchSerializationTests {
```

Comment:
Nit: Shouldn't these changes be in a different PR?

Reply:
I was poking around at all the other model stuff anyways